catkin_tools/verbs/catkin_build/cli.py

```diff
@@ -215,6 +217,7 @@ def main(opts):
             quiet=not opts.verbose,
             interleave_output=opts.interleave_output,
             no_status=opts.no_status,
+            status_update_rate=opts.status_update_rate,
```

Comment:
Also, we should put a limit on it based on the update loop rate, which is currently running at 10Hz:

https://github.com/catkin/catkin_tools/pull/141/files#diff-2482c854781a7d39252bf95c9201c631R521

Because as is, if the user puts `--status-rate=20` it will still only run at 10Hz.